### PR TITLE
Copy Files Portfolio to do review

### PR DESCRIPTION
**Pull request by ManarOmari**
pull request to review my code by myself

### CODE REVIEW

**Review by ManarOmari — COMMENTED** [review]
.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+        <h2>
+          <a href="#">My Logo</a>
+        </h2>
+        <img src="/assets/images/Icon-Menu.jpg" alt="Icon menu" />
+      </div>
+      <div class="header-wrapper">
+        <div class="text">
+          <h2>I’m Mariane <br />Glad to see you!</h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
```
Comment (ManarOmari) [inline index.html:25]: - [ ] to best practice , wrap div to section , <section> tag means that the content inside relates to a single theme ,It is recommended to use the <div> element as a last option and use other various tags such as <main> and <article>.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+          <div class="list-title">
+            <div class="list-icon"></div>
+            <h3>Frameworks</h3>
+          </div>
+          
+        </li>
+
+        <li class="list">
+          <div class="list-title">
+            <div class="list-icon" id="circle"></div>
```
Comment (ManarOmari) [inline index.html:268]: - [ ] Notice that you missed  icon -arrow 
🔽 for language and ▶️  for framework and skills

File: index.html [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+          <div class="work-buttons">
+            <ul>
+              <li class="html"><a href="#">html</a></li>
+              <li class="css"><a href="#">css</a></li>
+              <li class="js"><a href="#">javascript</a></li>
+            </ul>
+          </div>
+          <div class="see-project-btn">
+            <a href="#">See Projects</a>
+          </div>
```
Comment (ManarOmari) [inline index.html:128]: - [ ] Please add style for buttons " Active, Hover and Pressed " , to make your project match Figma design .

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @ManarOmari, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Required Changes ♻️
- [x] You should make a review of the past project and write at least 2 comments in the review.
- [x] You should use GitHub inline comments feature at least once to leave a review comment
- you can refer to [this lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-use-github-features) for how to make a good review

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ManarOmari — COMMENTED** [review]
change require ♻️ 
Please check comments below and make all require changes .

Please,  done all them 
Thanks 😁

**Review by ManarOmari — COMMENTED** [review]
Good Job 👍 
✔️  No lintral error 
✔️  Professional README file

**Comment by ManarOmari** [comment]
@omarsalem7  please check comment again . 😄

File: index.html (hunk 1 above)
Comment (danmainah) [reply index.html:25]: - [x] Kindly make sure your comment is polite and has an explanation of why the changes need to be made.

File: index.html (hunk 3 above)
Comment (danmainah) [reply index.html:128]: - [x] Kindly make sure to add an explanation of why the changes need to be made this not only helps the developer understand more but also adds more knowledge.

File: index.html (hunk 2 above)
Comment (danmainah) [reply index.html:268]: - [x] Kindly refer to the previous comment on the change to implement even here.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @ManarOmari, 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️  Good use inline comments 
 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ManarOmari** [comment]
@danmainah  please can u check the comments again ?

**Comment by ManarOmari** [comment]
> Hi @ManarOmari,
> 
> Good job so far! 👏 You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there!
> 
> you-are-near [ ](https://camo.githubusercontent.com/24e9f6bcfc9c66d6f90ce4fc937ec10b7ba92a648dadb764fb2c37246d36fbce/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f53535231374b36795a5776474359726655682f67697068792e676966) you-are-near [ ](https://camo.githubusercontent.com/24e9f6bcfc9c66d6f90ce4fc937ec10b7ba92a648dadb764fb2c37246d36fbce/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f53535231374b36795a5776474359726655682f67697068792e676966)
> 
> ## Required Changes ♻️
> * [x]  You should make a review of the past project and write at least 2 comments in the review.
> * [x]  You should use GitHub inline comments feature at least once to leave a review comment
> * you can refer to [this lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-use-github-features) for how to make a good review
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

please can you review my comments ?

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @ManarOmari 👋

Your project is complete! ✔️ 🏁. This is the First Project 1️⃣ You get Approved ✔️! in microverse 🌐.

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.


Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ManarOmari** [comment]
Thank you very much for this great support

**Pull request merged**